packages/server/src/server.ts

```diff
@@ -46,6 +47,8 @@ const cors = require('koa-cors');
 const app = new Koa();
 const rpc = koaJsonRpc();
 
+const REQUEST_ID_STRING = `Request ID`;
+const REQUEST_ID_MAX_VALUE = 100000000;
```

Comment:
I think this is an okay start.
In future with more request we'll want to explore this range or consider alpha numeric values to avoid collisions.
Knowing the time range of an in issue will likely avoid collision confusion so this isn't a biggie

Reply:
We could use randomUUID from crypto module. This will generate something like this 
`[2022-08-05 21:24:52.640 +0000] INFO (rpc-server/3975 on Georgis-MacBook-Pro-2.local): [Request ID:48f98bb9-45a2-4206-a0be-75482b8abe53] [POST] eth_sendRawTransaction: 200 571 ms 
`

Reply:
Yes, much better.
Make it so :)